client/my-sites/upgrades/cart/popover-cart.jsx

```diff
@@ -10,7 +10,7 @@ var React = require( 'react' ),
  */
 var CartBody = require( './cart-body' ),
 	CartMessagesMixin = require( './cart-messages-mixin' ),
-	CartButtons = require( './cart-buttons' ),
+	CartButtons = require( './cart-buttons' ).default,
```

Comment:
Tempting to either move this to its own `import` line or ES6-ify all of this to `import`s. The `default` is quite the eye sore.